e2e/edit-frame.test.ts

```diff
@@ -202,14 +203,133 @@ test('basic edit frame', async ({ browser }) => {
 	await page.getByText('Save').click()
 	await expect(page.getByText('Live Updates Paused')).toBeHidden()
 	try {
-		await expect(page).toHaveScreenshot('5-restored.png', { fullPage: true })
+		await expect(page).toHaveScreenshot(`${testPrefix}-5-restored.png`, { fullPage: true })
 	} catch (error) {
 		console.warn(error)
-		failedScreenshots.push('5-restored.png')
+		failedScreenshots.push(`${testPrefix}-5-restored.png`)
 	}
 
 	if (failedScreenshots.length > 0) {
 		console.log(`Failed screenshots: ${failedScreenshots.join(', ')}`)
 		throw new Error(`Failed screenshots: ${failedScreenshots.join(', ')}`)
 	}
 })
+
+
+const createDeleteFrameConfig = {
+	components: [
+		{
+			name: 'base-1',
+			api: 'rdk:component:base',
+			model: 'rdk:builtin:fake',
+			attributes: {},
+			frame: {
+				parent: 'world',
+				translation: { x: 0, y: 0, z: 0 },
+				orientation: {
+					type: 'ov_degrees',
+					value: { x: 0, y: 0, z: 1, th: 0 },
+				},
+			},
+		},
+		{
+			name: 'no-frame',
+			api: 'rdk:component:base',
+			model: 'rdk:builtin:fake',
+			attributes: {}
+		},
+	],
+}
+
+test('create and delete frame', async ({ browser }) => {
+	const testPrefix = 'CREATE_DELETE'
+	await viamClient.appClient.updateRobotPart(
+		testConfig.partId,
+		testConfig.name,
+		Struct.fromJson(createDeleteFrameConfig as unknown as JsonValue)
+	)
+	const failedScreenshots = [] as string[]
+	const context = await browser.newContext()
+	await context.addCookies([
+		{ name: 'MOTION_TOOLS_EDIT_FRAME', value: 'true', domain: 'localhost', path: '/' },
+	])
+	let page = await context.newPage()
+	page.on('console', (message) => {
+		console.log(`[${message.type()}] ${message.text()}`)
+	})
+	await page.goto('/')
+	await expect(page.getByText('World')).toBeVisible()
+
+	// SETUP CONFIG
+	await expect(page.getByTestId('icon-robot-outline')).toBeVisible()
```

Comment:
We should be avoiding test ids altogether - they're an antipattern masking a lack of accessibility. I can make a quick PR before this gets merged to do an accessibility pass for motion tools since that hasn't yet been done and then we can replace these :)